postcard-bindgen-core/src/code_gen/js/des.rs

```diff
@@ -41,17 +41,29 @@ fn gen_des_function_for_type(container: Container) -> Tokens {
     }
 }
 
-pub fn gen_deserialize_func(defines: impl Iterator<Item = Container>) -> Tokens {
+pub fn gen_deserialize_func(defines: impl Iterator<Item = Container>, esm_module: bool) -> Tokens {
     let body = defines.map(gen_des_case).join_with_semicolon();
-    quote!(
-        module.exports.deserialize = (type, bytes) => {
-            if (!(typeof type === "string")) {
-                throw "type must be a string"
+    if esm_module {
```

Comment:
TBH this if statement feels a little unnecessary, is there a good way to just pass a string into these `quote!` macros so I could just throw in either `modules.exports` or `export const `?

Reply:
I added an example to `ser.rs`:)